httpie/item.go

```diff
@@ -19,7 +19,7 @@ type Item struct {
 }
 
 func (i *Item) String() string {
-	return fmt.Sprintf(`"%s%s%s"`, i.K, i.S, i.V)
+	return fmt.Sprintf(`'%s%s%s'`, i.K, i.S, i.V)
```

Comment:
could you show me an example why you changed this?

Reply:
@dcb9 Single quotes are _generally_ better for raw strings IMO since you won't have to escape characters like `$`. Consider this:

    http get 127.0.0.1:8000/hello 'x-foo:$bar'

Right now it is converted to

    curl --header "x-foo: $bar" '127.0.0.1:8000/hello'

Which treats `$bar` as a variable even though it was a literal in first command. Ideally we should keep the quotation style as is (use single quotes if they're used in original command, double quoptes otherwise).

Reply:
@notpushkin SGTM
@yetingsky could you fix all the tests around this change.
you can simply run 
```shell
$ go test ./...
```

Reply:
fix all tests, PLTA @dcb9

Reply:
@yetingsky thanks a lot